### PR TITLE
Add parameter to pg_drop_subscription to specify if drop replication slot together

### PR DESCRIPTION
**Pull request by 0xlianhu**
Reason to add this change:

1. If lost contact to migration source, would not be able to drop subscription if dropping replication slot and subscription in one transaction
2. Prune has already handled replication slots after dropping the subscription

[BF-2380]

NOTE: Hannu mentioned that there is one use case that uses `aiven_extras` without using Aiven's prune

[BF-2380]: https://aiven.atlassian.net/browse/BF-2380?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: sql/aiven_extras.sql [hunk 1]
```diff
@@ -171,7 +171,8 @@ $$;
 
 DROP FUNCTION IF EXISTS aiven_extras.pg_drop_subscription(TEXT);
```
Comment (jankatins) [inline sql/aiven_extras.sql:172]: I would expect that the next version now also needs a 

```sql
DROP FUNCTION IF EXISTS aiven_extras.pg_drop_subscription(TEXT, BOOLEAN);
```

Might be nice to already add it now?

Comment (0xlianhu) [reply sql/aiven_extras.sql:172]: Thanks!

**Review by jankatins — COMMENTED** [review]
LGTM

**Review by alexole — APPROVED** [review]

**Comment by alexole** [comment]
@jankatins hey, thanks for still helping us :)

**Pull request merged**